src/constants/index.js

```diff
@@ -18,6 +18,8 @@ export const LIST_NAVIGATION_DIRECTIONS = {
     DOWN: 'ArrowDown',
 };
 
+export const CROSS_CHAR = '×';
```

Comment:
maybe we should have it in a separate file with commonly used chars. Also, it makes sense to replace current occurrences with this constant (e.g. in clear button in FieldWrapper)